custom_components/roborock/api/api.py

```diff
@@ -176,8 +164,8 @@ async def on_message(_client, _, msg, __=None):
                             if isinstance(decrypted, list):
                                 decrypted = decrypted[0]
                             await queue.async_put((decrypted, None), timeout=QUEUE_TIMEOUT)
-            except Exception as exception:
-                _LOGGER.exception(exception)
+            except RoborockException as ex:
```

Comment:
In these scenario what is going to be raised will not only be a RoborockException. And if an exception is not treated here, the thread that process messages will die